pkg/networkservice/chains/endpoint/combine_test.go

```diff
@@ -310,13 +313,15 @@ type testEndpoint struct {
 	networkservice.MonitorConnectionServer
 }
 
-func newTestEndpoint(ctx context.Context, name string) *testEndpoint {
+func newTestEndpoint(ctx context.Context, mechanism, name string) *testEndpoint {
 	e := new(testEndpoint)
 	e.NetworkServiceServer = next.NewNetworkServiceServer(
-		updatepath.NewServer(name),
 		begin.NewServer(),
 		metadata.NewServer(),
 		monitor.NewServer(ctx, &e.MonitorConnectionServer),
+		mechanisms.NewServer(map[string]networkservice.NetworkServiceServer{
+			mechanism: updatepath.NewServer(name),
```

Comment:
Have you tried using null.NewServer() here and leaving updatePath at the previous place?

Reply:
Yes, I have. It didn't work